### PR TITLE
docs: add tip about harmless vulnerabilities

### PR DESCRIPTION
**Pull request by ilg-ul**
## Pre-flight checklist

- [x] I have read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#pull-requests).

## Motivation

Attempt to minimise support requests asking if the reported `npm install` vulnerabilities are harmful.

Feel free to adjust the message if necessary.

## Related issues/PRs

- https://github.com/facebook/docusaurus/discussions/10525#discussioncomment-10775852

### CODE REVIEW

**Comment by ilg-ul** [comment]
Any idea what might be wrong with the failed test?

**Comment by Josh-Cena** [comment]
No idea, but definitely not caused by this PR.

**Comment by slorber** [comment]
Netlify fails randomly lately for unknown reasons. You can trigger it again with an empty commit and it should work.

**Comment by ilg-ul** [comment]
>  ... empty commit and it should work.

indeed.

File: website/docs/installation.mdx [hunk 1]
```diff
@@ -163,6 +163,12 @@ Then, in the directory containing `package.json`, run your package manager's ins
 npm install
 ```
 
+:::tip
+
+`npm install` may report several vulnerabilities and recommend running `npm audit` to address them. Typically, these reported vulnerabilities, such as RegExp DOS vulnerabilities, are harmless and can be safely ignored.
```
Comment (slorber) [inline website/docs/installation.mdx:168]:
```suggestion
`npm install` may report several vulnerabilities and recommend running `npm audit` to address them. Typically, these reported vulnerabilities, such as RegExp DOS vulnerabilities, are harmless and can be safely ignored. Also read this article, which reflects our thinking: [npm audit: Broken by Design
](https://overreacted.io/npm-audit-broken-by-design/).
```

**Comment by ilg-ul** [comment]
Thank you!

**Pull request merged**